ibutsu-server-openshift.yaml

```diff
@@ -1024,3 +1154,15 @@ parameters:
   displayName: Backend Route
   description: The URL of the backend of the Ibutsu server
   value: ibutsu-api.example.com
+- name: BACKUP_VOLUME_CLAIM
+  displayName: Backup volume claim
+  value: database-backup
+- name: BACKUP_STORAGE
+  displayName: Backup storage
+  value: 30Gi
+- name: BACKUP_KEEP
+  displayName: Number of backups to keep
+  value: '5'
+- name: BACKUP_SCHEDULE
+  displayName: Cron-like schedule to run backup
+  value: '50 * * * *'
```

Comment:
Will backing up this often impact performance?

Reply:
Heh, whoops, that's from the example. I need to fix that.